source/includes/oai-pmh/_root.md

```diff
@@ -143,6 +187,39 @@ correct community identifier.
 [See example](https://zenodo.org/oai2d?verb=ListRecords&
 metadataPrefix=oai_datacite&set=user-cfa)
 
+
+```python
+# Fetch a couple of records from the OAI Set of the "cfa" community
+>>> records = sickle.ListRecords(metadataPrefix='oai_dc', set='user-cfa')
+>>> record = records.next()
+>>> record.sets  # ...or whatever one can do to inspect what sets a record is in
+```
+
+### Harvesting with a different metadata format
+
+```python
+# Community harvest using "marc21" metadata format
+>>> records = sickle.ListRecords(metadataPrefix='marc21', set='user-cfa')
+>>> record = records.next()
+>>> record.raw  # ...or whatever showcases the metadata format
+<...>
+```
+
+
+### Even more selective harvesting
+
+```python
+# Selecting harvesting using "from"
+records = sickle.ListRecords(**{'metadataPrefix': 'oai_dc', 'set': 'user-cfa', 'from': '2019-01-01' })
+
+records.next()
+<Record oai:zenodo.org:7661>
+
+records.next()
+<Record oai:zenodo.org:6738>
+```
+
+### Other questions on harvesting
 If you need selective harvesting and your use case is not supported by above
 sets, please [contact us](http://about.zenodo.org/contact/) and we may possible
```

Comment:
```suggestion
sets, you can [contact us](http://about.zenodo.org/contact/) and we can try
```

Reply:
I would point to the support form actually, i.e. <https://zenodo.org/support>